src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/SwaggerSamClient.scala

```diff
@@ -119,6 +119,11 @@ class SwaggerSamClient(samBasePath: String, cacheExpiryTime: FiniteDuration, cac
     hasActionOnResource(notebookClusterResourceTypeName, getClusterResourceId(googleProject, clusterName), userEmail, googleProject, action)
   }
 
+  def getUserProxyFromSam(userEmail: WorkbenchEmail): List[WorkbenchEmail] = {
+    val samAPI = samGoogleApi(getAccessTokenUsingPem(leoEmail, leoPem))
+    List(WorkbenchEmail(samAPI.getProxyGroup(userEmail.value)))
```

Comment:
If Sam only every returns 1 email, maybe this should return `WorkbenchEmail` instead of `List[WorkbenchEmail]`.